types/stpadoptions.pp

```diff
@@ -0,0 +1,15 @@
+# Matches Pad Options Keys And Values
+type Etherpad::Stpadoptions = Struct[
+  {
+    Optional['noColors']         => Boolean,
+    Optional['showControls']     => Boolean,
+    Optional['showLineNumbers']  => Boolean,
+    Optional['useMonospaceFont'] => Boolean,
+    Optional['userName']         => Variant[Boolean, String],
```

Comment:
http://etherpad.org/doc/v1.6.6/#index_username suggests this is always a string. Do you know how it behaves when you enter a boolean?

Reply:
https://github.com/voxpupuli/puppet-etherpad/blob/e2ef02667095c2967ffc7628b2910c74fb89006d/templates/settings.json.epp#L57-L70

I only reproduce what was in the module and adding the possibility of putting a  `String`  as suggested in the Etherpad documentation.

Reply:
What do you mean about this ? @ekohl